### ISSUE TITLE
Connect failed: Host 'ip' is not allowed to connect to this MariaDB server

### ISSUE THREAD

**Issue by stjeebee** [Cyberbyte-Studios/CyberWorks#75]
Hello,

Whenever I attempt to finish the installation of Cyberworks I always get: 'Connect failed: Host 'ip' is not allowed to connect to this MariaDB server' when pressing setup. All the setup information I gave was correct, but I'm still getting the error. Anyone know where I should begin? Thanks.

**Comment by schasching** [comment]
make a neu user in the database --> name:support host:the ip from the server password: your pw --> use this user and it works :-D

**Comment by MightySCollins** [comment]
How is your database setup? It sounds like you have not allowed the database to be accessed externally by the webserver.

**Comment by stjeebee** [comment]
I got it working properly. But, now the players,vehicles,houses tabs aren't working. It shows on the dashboard that there's players and vehicles, but when I open the specific tabs it shows that it cannot find anything. I saw someone with the same issue and there was a supposed fix but the fix was actually in my version, and it didn't work. I use AltisLife 3.4.8. Gangs show up fine. Sorry for all the questions.
